medical_diagnostic_report/models/medical_diagnostic_report_template.py

```diff
@@ -85,3 +84,11 @@ def _generate_report_observation_vals(self, encounter):
             "blocked": self.blocked or self.template_id.item_blocked,
             "sequence": self.sequence,
         }
+
+    _sql_constraints = [
+        (
+            "concept_id_uniq",
+            "UNIQUE (concept_id)",
```

Comment:
Concept must be unique for each template. Two templates could use the same concept, isn't it?

```
            "UNIQUE (concept_id, template_id)",
```